log10/load.py

```diff
@@ -107,49 +108,53 @@ def get_session_id():
     return id
 
 
-# Global variable to store the current sessionID.
-sessionID = get_session_id()
-last_completion_response = None
-global_tags = []
+#
+# Context variables
+#
+session_id_var = contextvars.ContextVar("session_id", default=get_session_id())
```

Comment:
Slight preference to not set a default here and force an explicit set.

Reply:
We need this to keep the existing behavior - where llm calls in outer most "context" i.e. no context, still gets grouped together